apps-rendering/src/components/Callout/styles.ts

```diff
@@ -112,13 +113,31 @@ export const calloutSharelink = (
 	`}
 `;
 
-export const supportingText = css`
-	${textSans.xsmall()};
-	color: ${neutral[46]};
+export const sharePopup = css`
+	${textSans.xxsmall()};
+	position: absolute;
+	transform: translate(0, 100%);
+	display: flex;
+	align-items: center;
+	background-color: ${neutral[100]};
+	color: ${neutral[7]};
+	font-weight: normal;
+	border-radius: 4px;
```

Comment:
I think in AR they usually use `rem` rather than `px`. Can we change the 4px here to the equivalent rem of it here?